code/modules/unit_tests/outfit_sanity.dm

```diff
@@ -8,6 +8,17 @@
 	outfit_item.on_outfit_equip(H, FALSE, ##slot_name); \
 }
 
+/// See #66313 and #60901. outfit_sanity used to runtime whenever you had two mergable sheets in either hand. Previously, this only had a 3% chance of occuring. Now 100%.
```

Comment:
This is not a good pattern, we can't rely on github always existing, or downstreams knowing what these numbers mean.
Comments should be enough on their own

Reply:
it literally gives an explanation of the problem in the github issues

Reply:
It's not really your fault since moth told you to do it, but I just really don't like linking anything github related to the code itself, because it's non permanent, and could just go poof someday